src/main/java/com/feedzai/commons/sql/abstraction/engine/impl/DB2Engine.java

```diff
@@ -444,7 +444,7 @@ protected MappedEntity createPreparedStatementForInserts(final DbEntity entity)
                         .setUpsert(psUpsert);
 
         } catch (final IllegalArgumentException e) {
-            logger.error("Returning entity without an UPSERT/MERGE prepared statement.", e);
+            logger.error("{} Returning an entity without an UPSERT/MERGE prepared statement.", e.getMessage());
```

Comment:
Not sure if this change is really an improvement as we are missing the exception's stack trace.